### DISCUSSION TITLE
Mushaf Madina Font

### DISCUSSION THREAD

**Discussion by abshuraim** [quran/discussions#6] [Q&A]
Assalamualaikum warahmatullahi wabarakatuh, I need Mushaf Madina or KFC V2 Font for use in videos, So that I simply apply font on Quran Text. Please help.

**Comment by yoursjerry** [comment]
https://github.com/mustafa0x/qpc-fonts/tree/da7957c879f96213acb163973e9b688c21ac7819/mushaf-v2

**Comment by yoursjerry** [comment]
> Assalamualaikum warahmatullahi wabarakatuh, I need Mushaf Madina or KFC V2 Font for use in videos, So that I simply apply font on Quran Text. Please help.

Tajweed fonts are available too on tarteel website https://qul.tarteel.ai
